src/hardware/motor.cpp

```diff
@@ -227,7 +227,7 @@ float StepperMotor::getMicrostepMultiplier() const {
 void StepperMotor::step(STEP_DIR dir, bool useMultiplier, bool updateDesiredAngle) {
 
     // Main angle change (any inversions * angle of microstep)
-    float angleChange = (this -> fullStepAngle) / (this -> microstepDivisor);
```

Comment:
It is an dispensable division

Reply:
Absolutely. The best part is that I wrote in support for it, just never actually implemented it :)